packages/HelpAndCancel/Microsoft.Bot.Components.HelpAndCancel.nuspec

```diff
@@ -11,7 +11,7 @@
         <projectUrl>https://github.com/Microsoft/botframework-components</projectUrl>
         <releaseNotes>First version of HelpAndCancel package</releaseNotes>
         <description>Dialogs to provide help and cancel utterance handling</description>
-        <tags>bf-component conversationalcore helpintent cancelintent preview</tags>
+        <tags>msbot-component msbot-content conversationalcore helpintent cancelintent preview</tags>
```

Comment:
Looking through the second order tags defined, msbot-content seems pretty generic in comparison to the others. Is there a reason we landed on this versus something like msbot-dialog?